arch/arm/boot/dts/imx28-brain.dtsi

```diff
@@ -375,7 +375,7 @@
 
 	backlight_display: backlight@0 {
 		compatible = "pwm-backlight";
-		pwms = <&pwm 0 50000>, <&pwm 1 50000>;
+		pwms = <&pwm 0 50000>;
```

Comment:
How about adding the GPIO_3_5 to the `enable-gpios`?
https://www.kernel.org/doc/Documentation/devicetree/bindings/leds/backlight/pwm-backlight.txt

Reply:
the GPIO_3_5 switches the mode of current limit. 
Therefore, controlling by `enable-gpios` will not turn off the backlight.
Also, since `pwm_backlight_power_off()` does not stop the PWM signal, the backlight would only dim.

pwm_backlight_power_off: https://github.com/torvalds/linux/blob/master/drivers/video/backlight/pwm_bl.c#L60

Reply:
I meant that the GPIO should be described somewhere in the device tree as long as it is related to the backlight. GPIO3_5 is necessary to be set high for ideal brightness while it does not turn on or off the backlight.